### PR TITLE
Introduce 2019 prospectus

### PR DESCRIPTION
**Pull request by Dzol**
# Introduce 2019 Prospectus

Material to give potential sponsors an idea of what EnthusiastiCon is about and to ask for their involvement.

Inspiration, this year, and last: https://starcon.io/prospectus/

### CODE REVIEW

**Comment by hellerve** [comment]
This looks good to me!

(Except that my name is the first in the list of organizers, and there is just no way to justify that; maybe we should order it by alphabet or something?)

File: supplementary/2019/prospectus.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+EnthusiastiCon is a free two day event in Berlin around enthusiasm for computing.
+2019 will be its third edition.
+It is one day of short talks at 10 minutes each followed by a second day of participant led workshops.
+We aim for diversity in our speakers and their material while aiming for a broad audience too by proactively reaching-out.
+We hope to open up conference speaking to people who'd otherwise be hesitant because of background or the material they want to present.
+Our only criteria are that the material is about computing and enthusiasm!
+
+## Last Year
+
+- How many attendees
```
Comment (Dzol) [inline supplementary/2019/prospectus.md]: @zormit How many attendees did we have last year?

File: supplementary/2019/prospectus.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+In 2018 we were humbled by the quality and refreshing variety of our talks.
+What's more is that we were able to make EnthusiastiCon free to attend which we believe is key to making it accessible and inclusive.
+We were able to do this because of the support we recieved from sponsors.
+
+* We had more than double the number of talks proposed than slots.
+* 60 attendees on the first day of talks.
+* 30 attendees on the second day of participant-led workshops.
+* Many were first time speakers.
```
Comment (Dzol) [inline supplementary/2019/prospectus.md]: These are placeholders - what were the exact numbers?

File: supplementary/2019/prospectus.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Our Team
+
+The 2019 organising team is Nasreen Abu-Hunaina, Veit Heller, Moritz Need, Jens Ohlig, Joseph Yiasemides.
+Inspired by !!Con in NYC and its other derivative conferences we liked the idea of holding a similar event in Europe.
+One of our organisers wrote a blog post in 2018 with some context surrounding the conference: https://blog.veitheller.de/Enthusiasticon.html
+
+## Sponsorship
+
+The way we approach sponsorship is directly through the event's costs.
```
Comment (Dzol) [inline supplementary/2019/prospectus.md]: are we all happy with this statement? We haven't spoken about this explicitly

File: supplementary/2019/prospectus.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+In return for sponsorship we can offer:
+
+- Logo + name on site
+- Logo + name on correspondance
+- Logo, mention, and acknowledgement at opening and closing
```
Comment (Dzol) [inline supplementary/2019/prospectus.md]: If there's something else let's add it here!

File: supplementary/2019/prospectus.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Event Costs
+
+- Keynote speaker (transport & accomodation)
+- Space (insurance)
+- Catering (lunch)
+- Coffee
+- Meet-up page or EventBrite
+- Web site
```
Comment (Dzol) [inline supplementary/2019/prospectus.md]: Is there anything else we should include?

File: supplementary/2019/prospectus.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+## Event Details
+
+* Date: 25 & 26 May 2019
+* Venue: TBD
+* Attendance: 30-60
```
Comment (Dzol) [inline supplementary/2019/prospectus.md]: can we list the venue here _now_? The attendance is the same as what we estimated for last year IIRC

Comment (Dzol) [reply supplementary/2019/prospectus.md]: considering our recent e-mails it seems fine to me list ThoughtWorks here 🙂

File: supplementary/2019/prospectus.md (hunk 3 above)
Comment (Dzol) [reply supplementary/2019/prospectus.md]: If we're both on the same page then I'm happy to think we _all_ are

File: supplementary/2019/prospectus.md (hunk 2 above)
Comment (Dzol) [reply supplementary/2019/prospectus.md]: I've been more conservative here; I'm surprised I couldn't find a source (e.g. the Meetup page) with realistic numbers

**Pull request merged**